packages/amplify-graphql-api-construct-tests/src/__tests__/restricted-field-auth-sql-only.test.ts

```diff
@@ -1,447 +0,0 @@
-import * as path from 'path';
```

Comment:
This was renamed to `packages/amplify-graphql-api-construct-tests/src/__tests__/restricted-field-auth/restricted-field-auth-gen2.test.ts` and pretty heavily refactored to add new describe blocks and reuse the test methods.